<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>650</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration Screen</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Camera URL</string>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bağlanacak olan kameranın rtfs uzantılı url'in girildidiği alandır. Ör: &amp;quot;&lt;span style=&quot; font-family:'Menlo'; font-size:9pt; font-weight:600; color:#008080;&quot;&gt;rtsp://admin:password@192.160.1.100:554/chID=13&amp;amp;steamType=main&lt;/span&gt;&lt;/p&gt;&lt;p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera Name</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="camera_url">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Yazılan URL'in doğru olup olmadığını kontrol eder dğru ise ana ekranı doldurur.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_widget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>521</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="camera_name">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dellBtn1">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Yazılan URL'in doğru olup olmadığını kontrol eder dğru ise ana ekranı doldurur.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Configuration Screen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>-7</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
